framework/lib/components/color-picker/color-picker.tsx

```diff
@@ -136,8 +136,22 @@ export const ColorPicker = ({
             <PopoverHeader
               color="text.default"
               sx={ heading }
+              display="flex"
+              alignItems="center"
+              justifyContent="space-between"
+              width="full"
             >
               Pick a color
+              { trigger.color !== 'unset' && (
+                <IconButton
+                  aria-label="Clear color"
+                  size="sm"
+                  height="95%"
+                  onClick={ () => handleSelect(null) }
+                  variant="ghost"
+                  icon={ <Icon as={ IRemoveSolid } /> }
```

Comment:
Also I'm a bit unsure about the x, it gives the impression of closing the modal as opposed to reseting selected color. How about a button that says reset?